workspace/src/main/resources/public/ts/delegates/actions/create.ts

```diff
@@ -16,6 +16,7 @@ export interface CreateDelegateScope {
     canCreateNewFolderShared(): boolean;
     openNewFolderView(): void;
     createFolder();
+    isSharedTree();
```

Comment:
pourquoi ne pas donner le type de retour?
```suggestion
    isSharedTree(): boolean;
```